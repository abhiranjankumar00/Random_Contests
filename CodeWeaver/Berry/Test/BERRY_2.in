18
42 928
5519 32474 14687 3985 236 375 12465 17504 18799 19801 19978 5530 21809 14854 11621 1952 19995 25750 26382 17118 9977 28786 23894 31405 30416 17756 4747 15224 7794 28336 6951 25630 27727 32745 32840 20120 12942 8806 9824 27612 861 12399 
255 136
22273 1925 371 30189 9123 9025 19347 22731 27529 23888 15973 6812 24453 15433 15089 31184 24476 3631 19712 12706 24167 20037 18839 11923 5049 7459 14014 26391 17027 32522 24709 11406 5620 21621 16201 2165 23012 16624 4916 31198 4722 29741 22787 10013 14060 2389 31756 22469 5104 28828 27007 19155 6353 5912 26860 15656 5180 16595 24069 13373 9244 18719 22438 24850 23738 15657 4114 12135 27433 1233 17024 21807 14049 3513 31190 30274 18090 11420 15540 12370 27836 10466 15028 8992 32292 9841 24304 18671 6505 25029 27589 13113 16496 29251 983 32674 18845 1738 17493 6442 27707 20620 21631 6566 27678 4124 3685 24433 10374 30919 9457 11103 4679 24104 27431 23282 19751 24567 8364 4817 136 2082 26501 28021 6983 3871 18210 550 3811 14976 24731 22566 17890 2504 14393 6700 22896 9597 14392 7699 19018 17534 20683 5332 16602 31639 22881 30386 6307 11091 5927 32692 15072 32785 3278 20568 3176 14426 30606 1670 29397 9586 7531 25283 13541 6701 20663 19879 14881 15364 19672 20881 28092 27240 20555 23705 18621 12375 6121 24262 18273 29740 8041 6274 19898 22971 19239 11594 16285 2717 24355 20138 20411 22729 26445 4969 18958 28516 29952 20225 23927 1499 21964 32081 10405 18734 10104 5782 3640 28559 27975 24472 5158 10795 2683 2593 21086 26952 17944 14771 13484 9872 15338 16506 5085 10668 24265 10368 27171 31509 28700 23840 9915 31015 4252 2237 25804 17272 20059 31412 2451 28775 21508 10638 3517 16632 7524 9595 30228 18143 21801 3836 16222 12506 16362 
264 97
16770 12612 22632 17520 19949 13032 276 23278 3380 20499 28369 16188 24571 16365 12538 25817 18197 11784 31399 20754 5869 12346 4396 24438 31268 20136 995 10891 1915 21028 4396 27839 28404 19641 23729 28065 564 4227 23279 20305 7212 2244 5618 20533 11256 10549 12127 21742 26375 26522 17336 20107 26237 29593 4828 20732 13269 8396 4557 28105 2251 14101 18870 9988 18041 7474 10285 3708 26560 4497 9397 25907 29417 25957 24455 20242 11916 28153 7570 28380 7601 8260 17856 32353 1846 2751 17275 14068 3185 8563 29296 7710 4885 4182 30610 2708 1626 2007 11106 30055 13144 14264 31086 27515 17588 27676 22808 13114 8575 225 17305 31369 13147 7983 19270 8443 22496 31057 14353 29253 318 30916 19794 25843 18979 4750 7368 12633 223 13777 2778 21707 28102 2208 26659 21777 22741 26385 17454 22549 4761 22676 23934 25567 1101 28547 21202 23235 2883 18170 31143 2009 9093 4840 32253 20398 9659 30310 12126 22841 7247 5192 7965 17169 29648 3570 29660 14557 2708 25710 5712 32011 25506 28603 6985 15045 206 24679 3820 13344 10278 32608 1347 8343 6285 8199 5113 20409 29094 30854 6494 25630 21236 21603 29873 15007 30271 21846 5023 20934 18745 13912 30983 17796 19041 17661 8448 13090 11560 1283 26141 6522 2386 12843 19440 2407 6560 5191 21196 14970 19884 22933 14543 22511 13076 27467 19488 6623 31194 28787 12918 32439 6155 29433 18906 14568 20639 23299 1709 9440 8609 13440 29420 23354 23952 29444 12979 13105 29700 10195 12800 18779 11668 24124 24506 24212 24312 4648 6642 9015 22379 29938 12136 17045 
349 685
29817 6737 17055 21540 14452 22615 3046 2690 2566 994 11718 10150 22931 4758 13021 2306 22225 23837 6594 22824 20258 21542 17908 3304 17953 1709 32017 15288 31866 23933 17481 22362 22784 12276 12817 1720 26617 5639 2076 10259 25788 7045 10631 15964 383 14496 16724 19938 24938 13225 11545 27694 30960 20507 32761 1835 18526 28379 12756 10123 9396 958 20155 12901 693 1100 15015 28195 30778 13252 23914 20123 26571 28768 19173 19580 2672 26073 15228 19436 19397 18734 26823 22127 11341 418 8842 11275 7688 25347 12264 28586 3912 23858 20691 29375 6080 17632 29639 25644 31710 6550 29624 26212 31323 27896 29257 20684 6160 20298 8766 8296 24075 25907 5684 23475 15842 2753 26511 28110 4167 10158 18869 15581 25053 6370 23930 32687 586 21304 22746 5789 2446 9093 22254 27572 18323 12247 25625 7702 20996 7674 6787 31901 9315 833 25288 5058 1622 386 11505 21181 11330 13667 13731 3054 17947 22996 11158 21628 28795 17629 18761 1852 15345 8602 14068 28372 16027 2948 18623 23328 25591 28401 27953 9684 28406 21215 24555 27235 25170 28653 29846 29984 17186 14506 3610 11683 27052 14098 5648 111 21626 25430 2171 3026 12720 22492 26546 5132 4764 1671 11425 691 32143 21304 1841 30037 20971 14229 773 8032 12926 21513 25703 831 27150 4619 20866 18118 32309 15268 7121 9936 25712 11307 15831 20006 3442 32340 24849 5027 15038 23401 32809 25494 21108 20438 31084 22632 9579 18667 9598 18047 23425 8158 17996 16225 31279 22031 25155 20509 14689 26256 24512 32487 6788 20709 29895 19458 30176 16648 16119 14819 29224 30897 6077 8339 6735 24019 30921 20565 1438 21098 1214 18313 17924 14708 2094 32720 4269 19773 1733 5745 21717 10976 10436 21499 13535 14091 7345 22316 5537 16386 16717 26386 14339 231 11207 12400 4296 27192 24111 29283 21163 21401 16941 24133 24238 31444 15347 11986 3759 29726 26992 14985 3644 28572 25031 1365 13736 30757 2303 5485 7006 3914 19452 3961 20342 19091 22702 24054 30086 22028 9864 26587 12475 32768 25396 6781 5672 12805 6021 19120 6493 31544 6317 12591 10044 
23 60
6576 1598 25352 25020 28952 141 14783 14054 26347 15200 16298 24638 7898 18413 3122 30295 29371 28165 2037 16168 20793 24567 24857 
314 481
29898 26034 6624 18198 1762 17193 25981 23156 871 24060 18215 30039 28772 26196 10091 13024 15873 1806 21446 111 25267 24861 7283 5154 1968 22057 1154 11416 22950 26236 14378 32251 20908 9759 15712 28733 2681 3392 4776 4769 30163 10199 12332 26750 1611 26762 7947 29504 8488 8909 6899 7599 27809 28749 29170 27764 20047 1423 5769 6104 11745 14804 6147 29392 20498 17985 29956 17207 24024 32828 31845 28506 27614 4889 12351 9401 5685 31990 24390 23564 1095 21219 26713 24197 26892 4167 23639 12373 21376 30079 29235 1137 25428 14791 331 7574 13795 5214 30197 16522 8041 24805 29787 7078 1135 25352 7562 18321 26506 28001 10949 17742 32581 28569 21279 7545 26716 12241 12105 1229 18475 13120 4008 10730 29541 6685 30890 4181 27913 29375 14683 7361 20882 18435 20531 13965 623 28398 4658 8845 9650 22419 18480 31501 19980 25911 13934 26899 24313 4532 17008 741 312 24603 11023 29643 15230 25245 1738 20758 875 22444 31837 16481 23229 8797 27029 21271 19652 8721 27746 15158 24598 11604 558 15698 20611 13317 12653 988 9547 19640 29045 14244 30607 12632 17552 413 32006 28433 14416 23008 10993 19820 16419 18420 1361 12476 7719 30797 28247 31674 20032 19218 2588 13420 10087 16420 2706 28747 10053 25691 30651 19364 7168 9235 18456 28735 8277 9959 23909 28479 7655 6332 14723 29257 18902 1026 32694 2024 20372 8502 5648 3888 7668 16299 31533 1698 2692 11147 9249 16976 16157 13592 30383 19778 9656 15672 13244 31255 18051 27441 26114 26351 19509 28593 10505 30857 8862 30993 23635 4265 25474 15578 5847 957 1447 14985 28338 4543 557 7055 26904 26205 25630 21660 9872 25688 30278 27677 8033 16426 837 2564 1095 11755 23522 13511 5995 9845 6050 24491 4498 1991 17306 11842 26517 5287 29569 4402 1041 6028 26889 8249 18783 11450 17775 21870 28615 30639 31057 3530 5104 18366 
271 323
29414 2459 28666 27031 12806 18065 22101 14859 9303 31194 32265 23939 19573 9008 4407 6569 19760 16052 11726 7617 15760 22751 31151 22074 32279 7866 16911 7810 29509 15954 10780 17695 18786 11110 11491 5628 9591 7268 30321 24320 26526 21396 22584 8933 25211 2194 23733 28232 11275 1537 15952 764 27548 20350 17796 19876 18717 25555 21055 22913 24984 28525 5260 13960 25853 785 14447 1464 16673 8906 2320 8106 32262 16678 20252 2410 12125 31393 28265 11141 8009 9570 1753 32611 5734 2341 7296 9943 25866 21369 2825 8693 4555 7211 18517 5924 20801 3586 14950 24948 11021 1964 9340 22833 28697 18681 28541 2744 26146 32839 20890 20361 19572 19518 6448 32158 7669 1469 7916 8443 13413 26131 9011 29575 12516 23923 6849 1818 19893 11682 11920 21975 32065 7411 29435 11452 31327 21877 6691 9335 11730 17515 9618 7291 3347 3580 23326 27732 22098 26924 272 16183 19895 30731 9606 19934 24557 28716 29974 27283 19937 10073 25780 5002 17863 22661 13014 32627 3423 13570 982 22608 8566 5621 7575 6276 28335 25094 1643 15917 1923 31580 2868 18612 31623 18888 28874 13475 13293 13632 21661 27062 31054 9360 156 20276 1703 21580 13909 31359 19873 6665 13661 11242 20501 26572 242 3532 13007 9155 10463 27091 28050 10291 6560 18586 15453 32690 14279 12157 13805 12694 19102 28844 11139 12973 31246 16766 13799 20015 25535 2824 5037 6896 2945 10479 15543 20698 10987 2682 20219 4763 9281 27115 14470 32667 7323 3600 17089 8753 324 18965 25332 3606 31841 12188 29704 17178 7379 9999 15761 10261 13662 25141 27660 12833 14090 23812 24164 3078 2761 
405 777
27836 20982 11867 15648 3176 13691 6255 11326 30012 7597 10251 5138 11274 6056 9005 6831 5734 11069 2227 16529 31660 26600 14712 29999 5957 15874 15552 22995 8229 30776 9174 9898 27032 26940 20723 26135 319 8634 9570 31309 28245 17038 23266 8477 30263 16180 3148 27148 2596 9344 12661 21161 16961 14223 4694 32030 2528 253 30058 13572 24833 6232 3416 13948 22297 17365 27090 27515 8979 16450 28605 4965 31893 17724 7314 2270 6788 6562 6404 6621 31434 19530 24291 9729 22359 25842 30272 30505 27749 8569 3538 2111 29697 4943 29336 11860 31431 12586 10944 22514 29165 12538 14280 22189 8835 15904 17379 26364 26104 24538 16030 27207 18401 9218 11459 1857 29942 25081 12157 9336 7005 10718 1726 28971 8236 13315 4830 28763 20009 25761 6533 14884 25044 14282 20938 26446 5799 10656 31644 23328 6556 28357 1633 24436 1264 4370 2782 18193 18726 17548 7255 2612 10064 2195 13559 4464 8330 26973 16754 8888 32455 30115 4072 9403 23248 15514 9741 11376 17289 26661 17031 2289 27254 18249 3479 16780 31103 23777 15612 14871 30192 18067 12052 29061 21700 4098 16709 3767 9251 17989 26011 12180 28201 24473 15282 11868 14569 19282 12503 20793 18354 15896 27498 6168 401 3663 1414 7350 8470 13527 23269 14590 4443 17910 28157 26514 7235 30354 10351 4776 19968 420 31394 17687 12760 489 7851 29105 19496 5083 625 17620 30121 21471 7438 13372 31730 3264 13176 16667 11070 10844 20576 8300 9584 17565 29530 15748 10836 22187 15167 19178 13608 30304 4047 21864 4955 8189 3739 3426 22676 27163 16163 14951 21494 13796 21051 3334 27299 12162 28851 17123 31614 13016 23862 20746 12342 26736 28019 2178 24448 11957 7803 26918 25763 9059 18538 5808 4154 24891 13642 9239 5685 17915 7988 10523 7458 21645 22968 22045 21163 1919 10880 31285 10591 10163 5423 12159 9141 31576 7823 8629 20557 24856 19180 8963 23418 11155 23483 790 23038 20749 4224 23671 12564 17251 28398 19479 16308 1092 22095 18219 11538 18845 25126 825 104 18005 24595 9331 26101 27451 23800 9797 17733 13373 3418 12842 3252 21978 5386 22723 22062 24282 29188 24874 18744 8764 3735 8836 24116 32112 16070 7125 26336 20166 11503 15897 32801 12327 31146 7312 20735 18947 4852 30553 15088 21130 1677 21570 24409 3359 21406 6623 7762 24073 32671 24793 29354 18028 29304 28553 20266 23672 24038 5061 24862 5089 13345 20218 23570 27279 29116 2171 13018 
344 880
16076 14873 25045 19048 11084 19188 11890 24650 2752 3217 32754 1540 13081 3628 13680 14301 8921 14927 7455 8646 21111 29347 28884 14806 13573 7687 31684 24310 11280 23775 19854 20446 29384 22446 7804 3648 17504 23528 18693 578 16824 15105 31895 29234 23098 8138 3476 8821 15362 19965 1691 12737 3905 32790 24328 1421 227 22373 31422 23237 24687 4706 22308 20161 21807 32064 2413 29816 28306 20677 29213 18929 5070 29528 18888 8699 5833 20572 27421 22046 28218 14780 32292 21713 23307 1991 26820 24557 32003 27455 10376 3938 6817 9644 31139 27689 8267 7158 28465 14202 6824 30515 24767 12862 12362 18366 20750 30919 24208 24201 12820 18111 32202 20660 10820 14174 20639 22878 16143 21984 8876 3515 2219 4663 14958 5990 21267 28157 8802 26873 20559 30124 3434 8412 16373 23057 23155 3674 12272 15870 31573 3322 14976 3136 17146 10764 25986 27088 24259 18562 1778 6222 32551 14081 21166 23722 29598 27704 30076 15909 15805 23921 24911 12664 24482 9061 11974 1106 27026 24917 14326 7432 2785 8790 20585 25224 10770 8429 15974 24720 18496 11483 22899 31773 13245 9018 29209 19493 19960 11465 26343 27272 26788 13957 2746 20805 32278 22649 16443 6121 18671 7894 28328 9550 32508 26950 13272 10625 13146 21630 30585 29700 9182 25106 17309 31286 30131 21134 1653 8719 9541 26050 8539 9488 11614 30045 26902 8058 10639 7614 31578 8127 30674 26886 3471 18998 7533 12402 6355 25874 29466 10216 1858 24725 23652 18383 18037 12161 3668 2340 25416 976 10573 10186 4833 17316 10805 24354 19694 19073 30421 21724 23341 19130 2435 9741 562 25385 32766 19073 10085 15651 4505 16978 6218 18091 12444 16825 22277 23782 23271 20261 4125 31440 24024 1930 7552 16428 4463 31996 15047 25793 20144 25468 20156 27587 9545 32786 28185 11152 5237 3612 31635 32157 19232 3813 2327 10927 13132 14201 24148 26446 19635 28856 18553 29814 6830 7594 5952 14481 10581 7186 27435 25762 28136 13126 14605 31171 1646 25917 9933 17105 7323 21994 9575 5734 28663 15560 27699 29959 19261 5303 14698 22431 
181 899
32262 31718 16940 2966 1745 18985 28345 19503 19853 19105 9356 25924 12769 28971 4000 31800 1975 13781 11350 11832 148 10472 7015 25702 16164 2086 29213 13111 18714 14672 23235 18713 19255 19947 6490 12378 25511 29764 18573 26693 26260 31154 25319 6542 20726 16923 27716 27120 23976 22713 20357 13076 28865 32286 17544 8373 19745 22390 21080 31234 18506 8973 16800 6723 28786 5649 4966 23744 22742 18328 20469 32734 32848 19684 16923 17292 1580 27366 29333 18141 8504 258 30664 26870 3354 4370 28082 12587 25062 2221 10783 613 2544 12201 288 7623 18911 24542 11705 14481 3843 27181 13018 10564 30697 19650 18576 28679 19244 7829 10621 7753 20621 21559 21638 25000 16454 16843 5835 13841 15516 32338 29148 809 2484 3117 12842 10476 30631 27356 18875 16276 13016 30220 17186 3773 28294 13777 24751 32710 7581 1072 15331 3962 11730 11232 31014 752 30958 25681 25532 25519 24682 20206 1526 1814 18237 22944 4517 9468 24050 10899 11476 16787 29949 31036 30391 28306 18767 30875 22750 10131 32552 14013 30894 19897 16173 5510 23826 1687 10682 
102 575
7278 589 5241 17672 9941 27892 17972 11530 123 3901 9447 18058 109 25172 2586 30149 26495 6923 12653 14759 25478 28724 30069 17734 21192 8270 20166 21168 23473 31333 18769 4838 11464 28211 25585 24360 30945 26507 3648 3955 915 30968 14511 6729 20972 6561 10966 25008 21348 31343 5128 20861 5904 13938 2603 13010 24856 25217 27334 22758 13419 16417 21084 2820 9393 6057 7673 20439 1154 32163 11018 32228 17365 1908 18801 13656 4795 13120 5081 9988 7245 1800 8775 19622 28130 11029 19186 30314 24198 922 25941 20912 2728 3477 17282 19728 14355 758 21945 32155 13856 2563 
126 339
26980 3625 29691 19286 23360 6208 20217 1484 9161 18465 20252 20819 27561 18022 21221 18804 31509 10746 1760 21658 6807 26064 27783 8044 28945 16085 18531 25178 767 24099 15220 9590 14076 21367 3027 30342 904 7141 24726 17751 10760 202 29940 23124 8974 25347 1706 6998 8155 18012 924 4798 17698 12619 15040 27842 783 13535 5176 24324 11912 12531 5310 11881 16307 10407 12357 18216 22968 15121 12253 6945 32712 5167 11992 14961 7006 27168 29483 23107 14276 23188 9495 15848 5850 15490 10760 13867 787 14873 9917 10974 704 18168 27817 25273 20625 18081 22169 32101 15704 30507 22680 21459 24337 29752 18191 13010 12576 26151 6449 9185 2521 12683 10902 14401 2373 15067 16422 2639 25891 4715 21561 26897 13870 7322 
399 175
25103 10910 2637 2071 7524 16881 13725 19085 16404 30187 14029 7139 18592 29916 17791 19844 6616 20769 14887 26468 10225 18925 17343 22244 31703 4426 9661 1270 196 13146 4674 6017 20498 739 29844 25095 28494 12042 16334 18821 13453 26207 5590 8090 17667 11034 23184 14572 25041 15524 1873 17096 30495 14938 21143 7176 8555 25485 32759 13626 7456 6349 27734 15358 21611 615 10844 5552 32586 32812 17190 27975 2534 18036 28475 22653 13602 19555 23490 22577 9588 28523 20466 14345 21492 28690 18121 4443 5554 11254 20737 30030 11597 14606 21312 572 6849 17970 29839 26715 7553 25822 20749 8605 7366 8243 17636 24788 27174 1342 25469 31770 4712 125 28400 22128 12439 13471 25535 3237 30549 11916 27843 27002 25714 6821 21350 25859 21181 24397 16403 13136 24090 16552 24299 9805 19925 29090 15546 26343 31030 13876 28713 18800 23126 31459 25397 16709 17200 1417 20703 14254 26875 9047 19237 18949 14776 30747 23858 13083 12295 21183 7634 25559 6462 11026 17960 3872 444 11468 8355 11517 15227 29644 25646 14587 13029 23971 20391 12240 1513 5206 6547 4157 4429 13920 2064 25401 26694 16375 17674 3915 27206 6544 17074 31544 27413 28581 17452 17374 8483 14262 24200 2400 19997 1678 1524 6019 21950 7111 31536 27992 14056 18064 11586 26652 6899 1966 25450 5468 28485 5313 17240 28764 15650 8393 1949 3770 13254 3560 31328 32847 6321 27754 2130 26198 4675 27394 32740 1488 136 7254 20663 22823 22613 13758 24403 25873 13460 11081 16669 21687 4572 19714 29827 8471 26693 18801 27002 18342 6010 14692 28202 21562 22799 6383 21479 16936 11541 20913 24676 30889 29932 14937 24355 26243 12377 12137 26416 229 4312 10554 10511 10642 22324 29504 22771 16449 12116 26768 2353 25412 32344 22943 2291 9065 22245 32589 6961 12062 1980 7970 6621 14552 30866 17623 22878 7349 18716 13592 24640 2400 26217 32136 18218 12341 2676 17926 21776 16549 23034 18223 11177 13020 5257 2595 6180 9634 2778 21900 23575 21375 20320 27613 696 25379 21793 32472 5711 5892 28869 31827 18923 8265 28652 6203 27476 6375 15045 14297 12610 26890 13277 8788 14796 7094 18996 6776 19747 26286 29078 17845 14464 27029 29992 1131 21851 26707 22594 8037 4318 1210 14716 28323 1945 15210 27660 23900 2423 10225 15183 28483 6641 21559 3237 11018 16513 5914 21378 3620 15706 18005 3010 28112 30294 16294 11837 26963 12336 
16 394
21404 20868 26611 10526 31083 10638 24754 9353 7535 9458 14714 13723 9017 18123 8069 28917 
155 349
1622 28848 12879 10395 7345 25135 30814 24070 24547 32640 14331 29639 20253 5533 27265 7384 13376 24182 23415 19127 32414 158 31087 19133 15109 14149 11821 19884 19706 13926 524 3463 19778 8001 3742 1446 12711 6412 1171 28937 8696 25062 18652 749 7762 27660 27920 3485 23198 23375 1806 2955 29531 30406 11236 16031 24478 25280 4591 22991 17494 14076 7748 26864 3285 17693 24911 27339 27788 9048 18106 20852 11798 20336 31888 23767 29561 32228 21923 916 19368 26983 11051 24077 16716 13968 431 18364 18115 16339 29993 10955 9065 23984 12389 10718 17943 32554 29610 7220 8426 9851 28316 4078 29254 7686 29025 18094 7962 31117 14595 24676 613 14244 23317 10341 22009 5989 5274 14526 6271 30310 32802 26567 5307 19176 30518 13307 28177 3665 8950 8710 22088 11229 32769 16672 20623 18320 22933 18842 9661 29039 32760 2486 28125 6383 15894 5308 21226 21678 21099 26698 28887 21089 28451 
25 742
30418 781 1408 28396 28106 22395 24701 18540 18479 8422 21169 32177 7929 27126 31605 12994 16363 23538 26441 26930 28953 1073 3956 4410 3496 
194 555
31979 12555 24533 17324 4724 5729 23608 28296 16907 18012 8778 25486 12677 8076 26150 22270 4723 25162 10124 18208 12617 4943 23083 4990 32704 27775 25376 5200 7800 20486 4026 17976 10722 29094 27008 24111 20061 10831 2933 22975 26527 10737 23706 24436 12119 31630 21429 909 16144 3987 10321 17654 26969 16488 5382 22139 11811 1350 3834 2102 20074 18249 5800 25929 17176 3270 1504 22995 6950 28382 32527 29980 1284 3411 25025 7187 1851 29851 26724 16379 11345 10803 21327 28001 9708 21302 17667 18257 11067 10542 13232 17097 5778 12586 5115 13934 1620 26897 8446 4734 1302 24373 26000 21766 32001 26667 27335 3669 31352 26331 12220 2480 507 23251 20110 28231 32076 10465 29354 27752 976 20960 1097 19187 567 3026 11578 26894 1376 28975 6881 2073 11382 14973 8576 29157 1632 13857 9397 20256 24152 29301 26948 16916 15321 19994 13335 6546 6591 25232 23171 12328 6314 32003 10189 19354 12451 6504 15341 4240 9730 5065 18321 24205 17806 989 18789 10272 11239 30428 12434 7006 7201 31633 2180 11919 10674 13489 4593 25896 14734 25963 27927 24826 7824 11328 14661 5032 21788 24597 19868 9310 8707 17287 
268 457
24000 6310 638 10185 14369 19551 11781 9062 21494 12572 10841 13573 30858 28493 17215 27172 27379 12033 23195 20205 12612 7755 25919 18360 5636 14059 10753 9168 32735 1834 30480 19834 29084 518 16776 15155 17999 6799 25411 1594 32208 11921 27379 19022 11627 1139 12909 12985 26841 24671 12104 22669 14191 29860 12711 22062 27030 12632 1363 13328 25211 14145 10509 4802 10760 29174 4130 13868 23314 7489 8767 22037 16852 14792 28039 22504 16028 13548 16099 21943 16580 18418 5273 5760 28132 21088 7969 23471 18758 13226 17460 10213 28570 26638 22565 25336 15083 27837 372 32571 25298 12091 31393 1023 27882 3080 9790 8537 387 8320 10568 12582 30875 29696 32695 18765 13066 15434 16212 17220 5116 16683 31947 23766 11940 9051 1793 22051 31791 2724 27815 12520 11767 32582 32784 29952 28666 29339 19775 7723 12170 15095 13893 5903 14918 1746 10698 25980 18760 2857 10217 27636 19616 28961 2369 25707 2817 18158 15432 30795 27067 30707 27875 6587 28905 4138 18996 545 24556 20167 24243 31205 26136 19480 11905 19833 21420 14000 4089 5014 22280 27588 29319 10440 6682 28373 1981 1068 21530 1167 675 18808 9079 29938 24911 15421 17425 14511 29878 30825 30120 5589 6850 5136 9876 28588 24229 3692 14007 19937 29548 12322 26289 24292 15575 16582 18258 31119 23144 13730 15701 25096 26810 15795 27979 21992 27924 20213 9173 9117 7998 6369 3606 20749 1427 32819 22120 5499 5976 2032 31054 10927 22860 19368 26283 14673 10368 28408 17201 25751 10089 9812 24091 14624 308 15619 12063 799 30553 15467 28830 20411 28019 6287 10783 7606 26447 1494 
