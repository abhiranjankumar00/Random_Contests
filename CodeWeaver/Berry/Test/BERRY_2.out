41 747
254 31091
263 44620
348 8462
21 6308
312 10740
270 13380
403 8399
342 6745
180 3511
101 2729
124 5761
398 37771
14 594
153 7736
22 556
192 5465
266 9941